aide/backend/backend_openrouter.py

```diff
@@ -0,0 +1,86 @@
+"""Backend for OpenRouter API"""
+
+import logging
+import os
+import time
+
+from funcy import notnone, once, select_values
+import openai
+
+from aide.backend.utils import (
+    FunctionSpec,
+    OutputType,
+    backoff_create,
+)
+
+logger = logging.getLogger("aide")
+
+_client: openai.OpenAI = None  # type: ignore
+
+OPENAI_TIMEOUT_EXCEPTIONS = (
+    openai.RateLimitError,
+    openai.APIConnectionError,
+    openai.APITimeoutError,
+    openai.InternalServerError,
+)
+
+
+@once
+def _setup_openrouter_client():
+    global _client
+    _client = openai.OpenAI(
```

Comment:
oh this is so neat that openrouter just uses the OAI client

Reply:
yeee